doc/guides/add_keycloak_to_openshift.md

```diff
@@ -0,0 +1,57 @@
+## Prerequisites
+
+- There is a Keycloak service available.
+- You have provisioned a sync application using our playbook.
+- oc tools must be installed
+- You must first create a client for your application in the Keycloak Administration Console, and then click on the `Installation` tab, select `Keycloak OIDC JSON` for `Format` Option, and then click on `Download`. Save the downloaded `keycloak.json`.
+
+## Connecting your application to Keycloak
+
+Any application that connects to keycloak must consume a keycloak.json file.  This guide demonstrates how to add a keycloak.json file to your sync application deployment.  It will create the environment variable `KEYCLOAK_CONFIG` that contain the path to keycloak.json.  It is still your application's responsibility to consume this file.  We have provided an [example project](../../examples/keycloak).
+
+## Create a Keycloak Secret
```

Comment:
OpenShift template will help to simplify stuff a lot here, however we still do apb :(